packages/readiness/bundle/config/upstream/rbac.yaml

```diff
@@ -0,0 +1,60 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: tanzu-readiness-manager
+  annotations:
+    kapp.k14s.io/change-group: "readiness.core.tanzu.vmware.com/serviceaccount"
+  name: tanzu-readiness-manager-sa
+  namespace: #@ data.values.namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    kapp.k14s.io/change-group: "readiness.core.tanzu.vmware.com/serviceaccount"
+  name: tanzu-readiness-manager-clusterrole
+rules:
+  - apiGroups:
+    - core.tanzu.vmware.com
+    resources:
+      - readinesses
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - core.tanzu.vmware.com
+    resources:
+      - readinesses/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - core.tanzu.vmware.com
+    resources:
+      - readinesses/status
+    verbs:
+      - get
+      - patch
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  annotations:
+    kapp.k14s.io/change-rule: "upsert after upserting readiness.core.tanzu.vmware.com/serviceaccount"
```

Comment:
i think you also need to add this annotation to handle the package deletion case 
```  
 kapp.k14s.io/change-rule.1: "delete before deleting readiness.core.tanzu.vmware.comserviceaccount"
```

Reply:
Done